lib/Scenario.js

```diff
@@ -21,7 +21,6 @@ module.exports = class Scenario {
     this.description = config.description || '';
     this.prerequisite = typeof config.prerequisite === 'function' ? config.prerequisite : null;
     this.config = config;
-    this.decided = false;
```

Comment:
Good catch. I'm wracking my brain to try to remember why I included this in the first place but due to the use of Promises I think it's wholly unneeded.